src/resolvers/projectResolver.test.ts

```diff
@@ -2964,3 +2966,192 @@ function similarProjectsBySlugTestCases() {
+    }).save();
+
+    const sampleProject: CreateProjectInput = {
+      title: String(new Date().getTime()),
+      categories: [SEED_DATA.CATEGORIES[0]],
+      description: 'description',
+      admin: String(user.id),
+      walletAddress: generateRandomEtheriumAddress(),
+    };
+    const accessToken = await generateTestAccessToken(user.id);
+    const accessTokenUser1 = await generateTestAccessToken(user1.id);
+
+    // Add project that user is its admin
+    const addProjectResponse = await axios.post(
+      graphqlUrl,
+      {
+        query: createProjectQuery,
+        variables: {
+          project: { ...sampleProject, title: String(new Date().getTime()) },
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      },
+    );
+
+    // Add projectUpdate with accessToken user1
+    const result = await axios.post(
+      graphqlUrl,
+      {
+        query: addProjectUpdateQuery,
+        variables: {
+          projectId: Number(addProjectResponse.data.data.createProject.id),
+          content: 'TestProjectUpdateFateme',
+          title: 'testProjectUpdateFateme',
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${accessTokenUser1}`,
+        },
+      },
+    );
+    assert.equal(
+      result.data.errors[0].message,
+      errorMessages.YOU_ARE_NOT_THE_OWNER_OF_PROJECT,
+    );
+  });
+  it('should can not add project update because of not found project ', async () => {
+    const user = await User.create({
+      walletAddress: generateRandomEtheriumAddress(),
+      loginType: 'wallet',
+      firstName: 'testProjectUpdateFateme',
+    }).save();
+
+    const accessToken = await generateTestAccessToken(user.id);
+    const projectUpdateCount = await ProjectUpdate.count();
+
+    const result = await axios.post(
+      graphqlUrl,
+      {
+        query: addProjectUpdateQuery,
+        variables: {
+          projectId: Number(projectUpdateCount + 1),
+          content: 'TestProjectUpdateFateme2',
+          title: 'testProjectUpdateFateme2',
+        },
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      },
+    );
+    assert.equal(
+      result.data.errors[0].message,
+      errorMessages.PROJECT_NOT_FOUND,
+    );
+  });
+  it('should can not add project update because of lack of authentication ', async () => {
+    const project = await saveProjectDirectlyToDb(createProjectData());
+    const result = await axios.post(graphqlUrl, {
+      query: addProjectUpdateQuery,
+      variables: {
+        projectId: Number(project.id),
+        content: 'TestProjectUpdateFateme2',
+        title: 'testProjectUpdateFateme2',
+      },
+    });
+
+    assert.equal(
+      result.data.errors[0].message,
+      errorMessages.AUTHENTICATION_REQUIRED,
+    );
+  });
+  it('should can not add project update because user not found ', async () => {
```

Comment:
It's good to have a test case like this, for when adding project update, but projected is invalid ( project doesn't exists)

Reply:
I wrote it. did you mean 
https://github.com/FatemeGhasemi/impact-graph/blob/5c54fee14ed94887ab25ad88a4824f5340bf2e6c/src/resolvers/projectResolver.test.ts#L3083

Reply:
Good job, thank you